libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -873,14 +873,48 @@ void AP_GPS::update(void)
 #endif
 }
 
+/*
+    determines if any gps's are degraded
+*/
+uint8_t AP_GPS::update_and_count_degraded_gps(void) {
+    uint8_t count = 0;
+    for (uint8_t i = 0; i < GPS_MAX_RECEIVERS; i++) {
+        if(state[i].status < GPS_OK_FIX_3D) {
+            _degraded_gps[i] = true;
+            count++;
```

Comment:
recommend `++count` for performance

Reply:
`i` in the for loop as well

Reply:
I'll swap, but for built in types I don't think there is a performance difference between pre and post increment.

I choose ++ to stick with ardupilot style but you are right thats probably not important.

Reply:
Got it, no worries